client/src/components/ListingsGroup.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, Row } from "react-bootstrap";
+// todo: import { Listing } from '';
+
+function ListingsGroup({ content, createdAt, id }) {
+  const [listings, setListings] = useState([{}, {}, {}]);
+
+  // todo:
+  // useEffect(() => {
+  //   fetch("api/listings/" + id)
+  //       .then(r => r.json())
+  //       .then(listing => {
+  //         setListings([...listings, listing]);
+  //       })
+  // });
+
+  return (
+    <Row className="px-2">
+      {listings.map((listingContent, i) =>
+        <div className="col-sm-4">
+          <div className="card">
```

Comment:
With react bootstrap, you don't have to use className for cards.
Something like this...might be cleaner
<Card style={{ width: '18rem' }}>
  <Card.Img variant="top" src="holder.js/100px180" />
  <Card.Body>
    <Card.Title>Card Title</Card.Title>
    <Card.Text>
      Some quick example text to build on the card title and make up the bulk of
      the card's content.
    </Card.Text>
    <Button variant="primary">Go somewhere</Button>
  </Card.Body>
</Card>

Check out https://react-bootstrap.github.io/components/cards/

Reply:
done